src/vs/workbench/contrib/notebook/browser/notebookBrowser.ts

```diff
@@ -878,7 +878,9 @@ export function getNotebookEditorFromEditorPane(editorPane?: IEditorPane): INote
 
 	const input = editorPane.input;
 
-	if (input && isCompositeNotebookEditorInput(input)) {
+	const isInteractiveEditor = input && isCompositeNotebookEditorInput(input);
+
+	if (isInteractiveEditor || editorPane.getId() === REPL_EDITOR_ID) {
```

Comment:
I wonder if we should identify REPL as `isCompositeNotebookEditorInput` in the future.

Reply:
that helper function only has 3 uses, and the other 2 wouldn't apply to the replEditorInput, so probably not worth it